Add test plan folder structure, template and readme (#99)
Gamify-IT/docs: test-plans/README.md

> # test plans

Reviewer: ```suggestion
# Test plans
```

> Contains all the test-plans of all our services, and the test plan templates.

Reviewer: ```suggestion
Contains all test plans of all our services, and the templates (warning: files inside this folder are in German).
```

Author: should we do it in german? we should discuss this in the daily

Reviewer: As discussed:
Luckily, the majority is for English, so our test plans will also be in English.

> ## folder structure

Reviewer: ```suggestion
## Folder structure
```

> the test-plans belong to their corresponding folder. For Example a test plan created for the Moorhuhn frontend:

Reviewer: ```suggestion
the test plans belong to their corresponding folder. For example, a test plan created for the Moorhuhn frontend:
```

> ```
> test-plans/minigames/moorhuhn/frontend/plans

Reviewer: ```suggestion
```bash
test-plans/services/moorhuhn/plans.[md,pdf]
```

```

> As soon as a tester has tested a feature according to a test plan, it has to be placed in the `executed` folder

Reviewer: ```suggestion
As soon as a tester has tested a feature according to a test plan, it has to be placed in the `test-log` folder
```

> ## naming convention

Reviewer: ```suggestion
## Naming convention
```

> test-plan files should be following this structure

Reviewer: ```suggestion
Test plan filenames should have the following structure:
```

> ```
> name-service-backend/frontend.md
> example:
> highscore-moorhuhn-frontend.md

Reviewer: ````suggestion
```bash
$ACTION-$SERVICE.[md,pdf]
example:
highscore-moorhuhn-frontend.md
````

```

> where `name` is the name of the feature the test plan covers

Reviewer: ```suggestion
where `$ACTION` is the name of the feature the test plan covers
```
↑ Also, as I understood it: There should basically be one test plan per service, each having multiple test cases.

executed tests should be named

> ```
> date-testername-name-service-backend/frontend.md
> example:
> 2022-08-01-MichaelLinder-highscore-moorhuhn-frontend.md

Reviewer: ```suggestion
```bash
$DATE-$TESTER-$ACTION_NAME-$SERVICE_NAME-testresults.md
example:
2022-08-01-michael_linder-highscore-moorhuhn-frontend.md
```
